packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1435,9 +1435,10 @@ Version
      }
 
 Model
-  = version:Version? ns:Namespace imports:Imports? body:SourceElements? {
+  = version:Version? decorators:Decorators? ns:Namespace imports:Imports? body:SourceElements? {
       const result = {
         $class: "concerto.metamodel@1.0.0.Model",
+        decorators: optionalList(decorators),
```

Comment:
I *think* that if you omit this when the list is empty then you won't have to update all the test JSON files... We also need to update the metamodel itself in models.accordproject.org repo (bump the version?) to add this as an optional attribute. I think the metamodel is also duplicated in @accordproject/concerto-metamodel repo?

Reply:
Given I've already updated all the test JSON files, I'll remember that for next time! I've updated the @accordproject/concerto-metamodel package (which is in this repo), but I will also update the models.accordproject.org repo in a separate PR.